libs/cart/src/drivers/magento/transforms/inputs/shipping-method.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+
+import { MagentoShippingMethodInput } from '../../models/inputs/shipping-method';
+import { DaffCartShippingRate } from '../../../../models/cart-shipping-rate';
+
+@Injectable()
+export class DaffMagentoShippingMethodInputTransformer {
+  transform(method: Partial<DaffCartShippingRate>): MagentoShippingMethodInput {
+    return {
+      carrier_code: method.carrier,
+      method_code: method.method_title
```

Comment:
I'm not sure if this is the correct transform for `method_code`.

Reply:
doesn't this just come from magento, then we give it back to them? Can't we just reverse the magento -> daffodil transform?

Reply:
Yeah and looking back at the [shipping rate transformer](https://github.com/graycoreio/daffodil/blob/develop/libs/cart/src/drivers/magento/transforms/outputs/cart-shipping-rate.service.ts) `method_code` is not preserved anywhere in `DaffCartShippingRate`. Do you think its appropriate to store `method_code` in the `id` field?

Reply:
Changed in [`b93ce21`](https://github.com/graycoreio/daffodil/pull/673/commits/b93ce2127ebd54b8dce07e820cd44da3a25343b6).